lib/public/services/lhcFill/computeLhcFillStatistics.js

```diff
@@ -100,7 +100,7 @@ function mergeOverlappingSegments(overlappingTimeSegments) {
  * @param {number} stableBeamsStart the timestamp (in milliseconds) of the stable beams start
  * @param {number} stableBeamsEnd the timestamp (in milliseconds) of the stable beams end
  *
- * @return {{efficiency: number, efficiencyLossAtStart: number, meanRunDuration: number, totalRunsDuration: number}} the LHC fill statistics
+ * @return {Object} the LHC fill statistics
```

Comment:
This function will be replaced by a class in a future PR, hence this precise doc-block is not necessary